src/user/user.controller.ts

```diff
@@ -4,19 +4,24 @@ import {
   Param,
   UseGuards,
   ParseUUIDPipe,
+  Patch,
+  Body,
 } from '@nestjs/common';
 import { UserService } from './user.service';
 import { AuthGuard } from 'src/auth/guards/auth.guard';
 import { User } from '@prisma/client';
 import { CurrentUser } from 'src/auth/decorators/current.user.decorator';
+import { UpdateUserDto } from './dto/update-user.dto';
 
 @UseGuards(AuthGuard)
 @Controller('')
 export class UserController {
   constructor(private readonly userService: UserService) {}
 
-  @Get('me')
-  findOne(@CurrentUser() loggedUser: User) {
-    return this.userService.findUnique({ id: loggedUser.id });
+ 
+
+  @Patch(':id')
```

Comment:
You should also consider some validation (guards) on this endpoint, so user cannot change another user's profile